version.php

```diff
@@ -23,7 +23,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2020110901;
+$plugin->version = 2021042900;
 $plugin->requires = 2020110900;
 $plugin->release = '3.10.1-opendata-001';
```

Comment:
please bump this version also

Reply:
What we have to do here, since I change the version with newest date + 00 means that stable version

Reply:
let's bump the release to `3.10.1-opendata-002`